udata/core/dataset/forms.py

```diff
@@ -79,6 +80,10 @@ class BaseResourceForm(ModelForm):
         _('Publication date'),
         description=_('The publication date of the resource'))
     extras = fields.ExtrasField()
+    schema = fields.StringField(
+        _('Schema'),
+        # choices=[schema['id'] for schema in ResourceSchemas.get()],
```

Comment:
Can't add this line here as `ResourceSchemas.get()` needs `current_app.config`. It isn't available there as it's not in the app context when called from here.

How to get around this?

Reply:
same as above, build the choices list at the module level.